src/site/content/en/blog/vr-comes-to-the-web/index.md

```diff
@@ -213,8 +221,7 @@ the information needed to render a single frame to the display. The
 Before doing anything else, I'm going to request the next animation frame. As
 previously stated, the timing of frames is determined by the user agent based on
 the underlying hardware. Requesting the next frame first ensures that if
-something during the callback throws an error I can ensure that the frame loop
-continues.
+something during the callback throws an error, the frame loop continues.
```

Comment:
```suggestion
the underlying hardware. Requesting the next frame first ensures that
the frame loop continues if something during the callback throws an error.
```
Suggested to avoid the "that if" construction, which makes the syntax a little harder to parse.